qemu/tests/driver_load.py

```diff
@@ -88,12 +88,19 @@ def service_operate(cmd, ignore_error=False):
         session.close()
 
     error_context.context("Try to log into guest.", logging.info)
+    print('fbq1111')
```

Comment:
@fbq815 Seems you forgot to delete the debug lines.

Reply:
Modified